TurkishFilter/sections/specific.txt

```diff
@@ -45,6 +45,7 @@ megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,maciz
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,hdfilmizledayi.com,fullhdfilmizlebaba.com###adStarter
 !
+incisozluk.com.tr##a[href="https://bit.ly/3BPGADW"]
```

Comment:
Can you show the element via opened devtools?
Please also check `incisozluk.com.tr##a[href^="https://bit.ly/"]`

Reply:
Sure, here: https://imgur.com/leUpxMD

Reply:
Ok, does it block only this? If so, please exchange your rule.